src/setupPagination.js

```diff
@@ -0,0 +1,52 @@
+import updateData from "./updateData.js";
+
+export default setupPagination;
+
+function updateSelectedButton(currentPage) {
+    const oldButtons = document.getElementsByClassName("selected");
```

Comment:
Why buttons instead of button?

Reply:
Considering there are two sets of the same buttons (one at the top of the page and the other at the bottom) there would also be in total two numeric buttons highlighted (selected).

So I though to put the name of the parameter in plural to reference these two buttons.

I guess following that logic I should have named the function in plural too (updateSelectedButtons). That's my bad..